File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/EventSearch.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

using System;


namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class EventSearch
    {
        public int EventId { get; set; }
        public string EventNumber { get; set; }
        public string MemberNumber { get; set; }
        public string MemberName { get; set; }
        public EventTypeEnum EventType { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime DateOfIncident { get; set; }
        public int RolePlayerId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/personEvent/event-search.ts
import { EventTypeEnum } from '../../enums/event-type-enum';

export class EventSearch {
    eventId: number;
    eventNumber: string;
    memberNumber: string;
    memberName: string;
    eventType: EventTypeEnum;
    createdDate: Date | string;
    eventTypeDescription: string;
    dateOfIncident: Date;
}
